hydroflow_plus_test/src/cluster/mod.rs

```diff
@@ -8,3 +8,9 @@ pub mod quorum;
 pub mod request_response;
 pub mod simple_cluster;
 pub mod two_pc;
+
+#[stageleft::runtime]
+pub mod docs {
+    #[doc = include_str!("../../../docs/docs/hydroflow_plus/consistency.mdx")]
```

Comment:
the mdx syntax won't look quite right

Reply:
Ah, this is meant to be hidden! Just to get it to be processed in doctests. Will add a `#[doc(hidden)]`